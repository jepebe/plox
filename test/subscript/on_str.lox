var s = 'abcdef';

print 'string subscript: s[1] ->' + (s[1] == 'b');