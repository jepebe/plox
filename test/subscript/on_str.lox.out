string subscript: s[1] ->true
